_posts/2022-12-06-poetry-ai.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "AI and poetry meet Trino"
+author: Martin Traverso, Marcos Traverso
+excerpt_separator: <!--more-->
+image: /assets/trino-light.png
+---
+
+Sometimes working in the depth of the query engine core, the planner, the
+optimizer, or some other tricky problem or code, you run out of steam and need a
+distraction. Your brain will quietly and in the background still work on the
+problem, while you get to read or do something interesting.
+
+That's when we found out more about GPT-3 and the recent improvements on the
+OpenAI project. So we had to try.
```

Comment:
"That's when" feels a little jarring. Maybe "one of these times," or "during one break, we-" to make it more clear you're transitioning from talking about a general phenomenon to a specific story.